src/travel-details-screens/utils.ts

```diff
@@ -106,9 +108,61 @@ export function significantWaitTime(seconds: number) {
   return seconds > MIN_SIGNIFICANT_WAIT_IN_SECONDS;
 }
 
-export function getLineName(leg: Leg) {
+/* NB this is the same function as in the bff. Keep in sync! */
+export function destinationDisplaysAreEqual(
```

Comment:
Doh! We needed this here as well. The strategy that was chosen was kinda based on that there was no such code in the app. Let's have a talk about this.

Reply:
I don't think the `destinationDisplaysAreEqual` function in itself is the issue, as it is basically just a deep equals for destinationDisplay objects. The challenge is the input we give it.